src/RA_Dlg_Memory.cpp

```diff
@@ -1470,8 +1477,12 @@ INT_PTR Dlg_Memory::MemoryProc(HWND hDlg, UINT nMsg, WPARAM wParam, LPARAM lPara
         }
 
         case WM_CLOSE:
-            EndDialog(hDlg, 0);
-            return TRUE;
+        {
+            // hDlg was invalid sometimes
+            if (hDlg)
+                EndDialog(hDlg, IDCLOSE);
```

Comment:
The Memory Inspector was open when RA thought I had no connection to the internet. After I closed it said "An invalid handle was specified", so I'm guessing is was invalidated early. nullptr doesn't just mean 0, It's any value not valid for pointer type.